gslib/tests/util.py

```diff
@@ -623,14 +623,19 @@ def MakeBucketNameValid(name):
   """Returns a copy of the given name with any invalid characters replaced.
 
   Args:
-    name (str): The bucket name to transform into a valid name.
+    name: The bucket name to transform into a valid name.
```

Comment:
May want to specify if it will be a byte string, unicode string, or either for clarity. That way if someone contributes but isn't aware of the string handling differences between Py 2 / 3, they'll know

Reply:
Should it mirror the return types--(unicode, str, bytes)?

Reply:
Yeah, that makes sense to me. (str (py2: bytes / py3: unicode)) or however you want to notate it

Reply:
Added type hint to mirror return types